examples/remotereconnection/public/index.html

```diff
@@ -34,22 +41,26 @@ <h5>State</h5>
             </div>
           </div>
         </div>
-            <div class="card">
-              <div class="card-block">
-                <h4 class="card-title">Remote View</h4>
-                <div id="p2-media"></div>
-                <button id="p2-simulate-reconnection" class="btn btn-primary btn-block">Simulate Reconnection</button>
-                <h5>State</h5>
-                <div id="p2">
-                  <div class="roomstate connected current">Connected</div>
-                  <div class="roomstate reconnecting">Reconnecting</div>
-                </div>
-              </div>
+      </div>
+      <div class="card">
+        <div class="card-block">
+          <h4 class="card-title">Remote View</h4>
+          <div class="card-body">
+            <div id="p2-media"></div>
+            <button id="p2-simulate-reconnection" class="btn btn-primary btn-block">Simulate Reconnection</button>
+            <h5>State</h5>
+            <div id="p2">
+              <div class="roomstate connected current">Connected</div>
+              <div class="roomstate reconnecting">Reconnecting</div>
             </div>
+          </div>
         </div>
       </div>
     </div>
   </div>
-  <script src="index.js"></script>
+<script src="index.js"></script>
+<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```

Comment:
@PikaJoyce Nitpick: Can you indent these `<script>` tags to align with the last `</div>` just above them?